<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>filter_wizard</class>
 <widget class="QWizard" name="filter_wizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>328</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Gill Sans MT</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MyBookMgr - Filter Booklist</string>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::ClassicStyle</enum>
  </property>
  <property name="options">
   <set>QWizard::NoBackButtonOnStartPage|QWizard::NoCancelButton</set>
  </property>
  <widget class="QWizardPage" name="filter_page1">
   <property name="title">
    <string>Filter Category</string>
   </property>
   <property name="subTitle">
    <string>How would you like to filter your booklist?</string>
   </property>
   <attribute name="pageId">
    <string notr="true">1</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Categories:</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="RadioBtn_author">
           <property name="font">
            <font>
             <family>Gill Sans MT</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Author</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="RadioBtn_rating">
           <property name="font">
            <font>
             <family>Gill Sans MT</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Rating</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="RadioBtn_genre">
           <property name="font">
            <font>
             <family>Gill Sans MT</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Genre</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="RadioBtn_series">
           <property name="font">
            <font>
             <family>Gill Sans MT</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Series</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="filter_page2">
   <property name="title">
    <string>Let's Narrow It Down...</string>
   </property>
   <property name="subTitle">
    <string>Choose an option from drop-down menu.</string>
   </property>
   <attribute name="pageId">
    <string notr="true">2</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QComboBox" name="comboBox">
      <property name="font">
       <font>
        <family>Gill Sans MT</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="maxVisibleItems">
       <number>2147483647</number>
      </property>
      <property name="insertPolicy">
       <enum>QComboBox::InsertAlphabetically</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
